ang/civicase/activity/panel/directives/activity-panel.directive.html

```diff
@@ -76,7 +76,7 @@ <h3 class="panel-subtitle clearfix">
           <!-- End - Activity Date -->
 
           <!-- Avatar -->
-          <span ng-if="activity.assignee_contact_name" contact-card avatar="true" contacts="activity.assignee_contact_name"></span>
+          <span ng-if="activity.assignee_contact_name" civicase-contact-card avatar="true" contacts="activity.assignee_contact_name"></span>
```

Comment:
Just a thought. Can we break it down into multiple lines, as it crosses the *80 letters in one line* limit.

Reply:
I would rather not fix any exiting code, but as the number of comments are low, I am working on the fixes :)